frontends/web/src/app.tsx

```diff
@@ -154,49 +153,49 @@ export const App = () => {
-                <div className="appContent flex flex-column flex-1" style={{ minWidth: 0 }}>
-                  <Update />
-                  <Banner msgKey="bitbox01" />
-                  <Banner msgKey="bitbox02" />
-                  <MobileDataWarning />
-                  <WCSigningRequest />
-                  <Aopp />
-                  <KeystoreConnectPrompt />
-                  {
-                    Object.entries(devices).map(([deviceID, productName]) => {
-                      if (productName === 'bitbox02') {
-                        return (
-                          <Fragment key={deviceID}>
-                            <BitBox02Wizard
-                              deviceID={deviceID}
-                            />
-                          </Fragment>
-                        );
-                      }
-                      return null;
-                    })
-                  }
-                  <AppRouter
-                    accounts={accounts}
-                    activeAccounts={activeAccounts}
+              <LightningProvider>
+                <Darkmode />
+                <div className="app">
+                  <AuthRequired/>
+                  <Sidebar
+                    accounts={activeAccounts}
                     deviceIDs={deviceIDs}
-                    devices={devices}
-                    devicesKey={devicesKey}
-                    lightningInactive={lightningConfig.inactive}
                   />
-                  <RouterWatcher />
+                  <div className="appContent flex flex-column flex-1" style={{ minWidth: 0 }}>
+                    <Update />
+                    <Banner msgKey="bitbox01" />
+                    <Banner msgKey="bitbox02" />
+                    <MobileDataWarning />
+                    <WCSigningRequest />
+                    <Aopp />
+                    <KeystoreConnectPrompt />
+                    {
+                      Object.entries(devices).map(([deviceID, productName]) => {
+                        if (productName === 'bitbox02') {
+                          return (
+                            <Fragment key={deviceID}>
+                              <BitBox02Wizard
+                                deviceID={deviceID}
+                              />
+                            </Fragment>
+                          );
+                        }
+                        return null;
+                      })
+                    }
+                    <AppRouter
+                      accounts={accounts}
+                      activeAccounts={activeAccounts}
+                      deviceIDs={deviceIDs}
+                      devices={devices}
+                      devicesKey={devicesKey}
+                    />
+                    <RouterWatcher />
+                  </div>
+                  <Alert />
+                  <Confirm />
                 </div>
-                <Alert />
-                <Confirm />
-              </div>
```

Comment:
With every provider there is always lots of lines that changed only due to indentation... for this reason I would prefer to merge https://github.com/digitalbitbox/bitbox-wallet-app/pull/2559 first, then master->staging-ln, and this PR